TicketBooking/Helpers/CustomerDB.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Data.SqlClient;
+using TicketBooking.Models;
+
+namespace TicketBooking.Helpers
+{
+    internal class CustomerDB
+    {
+        public static Customer Get(string customerId)
+        {
+            SqlConnection connection = new SqlConnection();
+            connection.Open();
+            SqlCommand command = connection.CreateCommand();
+            command.CommandText = "Select * From CustomerId Where CustomerId=" + customerId;
```

Comment:
1. can we avoid the Dynamic sql statement, to avoid SQL injection
2. can we check customerid is null orEmpty